.goreleaser.yml

```diff
@@ -0,0 +1,53 @@
+# This is an example goreleaser.yaml file with some sane defaults.
+# Make sure to check the documentation at http://goreleaser.com
+before:
+  hooks:
+    - dep ensure -vendor-only
+builds:
+- binary: fluxctl
+  main: ./cmd/fluxctl/
```

Comment:
Will this inject version at build time? see https://github.com/weaveworks/flux/blob/master/Makefile#L33

Reply:
default is `-s -w -X main.version={{.Version}} -X main.commit={{.ShortCommit}} -X main.date={{.Date}}`